Yarn warnings should be logged as Maven warnings too

As in #343 yarn warnings should be logged as Maven warnings too instead of errors.

```
[INFO] --- frontend-maven-plugin:1.3-SNAPSHOT:yarn (yarn install) @ frontend ---
[INFO] Running 'yarn ' in C:\dev\frontend
[INFO] yarn install v0.16.1
[INFO] info No lockfile found.
[INFO] [1/4] Resolving packages...
[INFO] [2/4] Fetching packages...
[INFO] info "fsevents@1.0.15" is an optional dependency and failed compatibility check. Excluding it from installation.
[ERROR] warning fsevents@1.0.15: The platform "win32" is incompatible with this module.
[INFO] [3/4] Linking dependencies...
[ERROR] warning Unmet peer dependency "eslint@>=1.6.0 <4.0.0".
[ERROR] warning Unmet peer dependency "babel-core@^6.0.0".
[INFO] [4/4] Building fresh packages...
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,7 +2,12 @@
 
 ## Changelog
 
-### 1.2-SNAPSHOT
+### 1.3
+
+* Fix `yarn` for Windows
+* Fix #515: Change "yarn warning" from ERROR to WARNING in log
+
+### 1.2
 
 * New goals `yarn` and `install-node-and-yarn` for Yarn support 
 

--- a/frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/InputStreamHandler.java
+++ b/frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/InputStreamHandler.java
@@ -53,8 +53,8 @@ final class InputStreamHandler extends Thread {
         return new LogLevelAgnosticLogger() {
             @Override
             public void log(String value) {
-                // fix #343
-                if (value.startsWith("npm WARN ")) {
+                // fix #343 and #515
+                if (value.startsWith("npm WARN ") || value.startsWith("warning ")) {
                     logger.warn(value);
                 } else {
                     logger.error(value);

